src/quest_manager/templates/quest_manager/tab_quests_approvals.html

```diff
@@ -23,14 +23,14 @@
         <th class="col-sm-1 col-xs-2" data-field="xp" data-sortable="true">XP</th>
         <th class="col-sm-1 hidden-xs" data-field="group_name" data-sortable="true">{% group_name %}</th>
         <th class="col-sm-2 col-xs-3 hidden-xs" data-field="user" data-sortable="true">User</th>
-        <th class="col-sm-3 hidden-xs" data-field="status" data-sortable="true">Status</th>
+        <th class="col-sm-3 hidden-xs" data-field="status" data-sortable="true" data-sorter="dateSorter">Status</th>
```

Comment:
See original issue for how to do this with only a couple lines of code using `data-sort-name`